ccd-definition/CaseEventToComplexTypes/UpdateReferenceNum-HelpWithFee-R2-CUI-nonprod.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "ID": "HelpWithFeesDetails",
+    "CaseEventID": "UPDATE_HELP_WITH_FEE_NUMBER",
+    "CaseFieldID": "hwFeesDetails",
+    "ListElementCode": "claimFee",
+    "EventElementLabel": "### Help with Fees type:  HwF Claim fee",
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "ID": "HelpWithFeesDetails",
+    "CaseEventID": "UPDATE_HELP_WITH_FEE_NUMBER",
+    "CaseFieldID": "hwFeesDetails",
+    "ListElementCode": "hearingFee",
+    "EventElementLabel": "### Help with Fees type:  HwF Hearing Fee",
```

Comment:
In other views we are not using `### `, I don't know if in your view it is required to appear in bold. Pls check.